src/components/ProjectDisciplines.js

```diff
@@ -65,12 +65,19 @@ export class ProjectDisciplines extends React.Component {
           text={'You must have an internet connection to use Zooniverse Mobile'} />
       </View>
 
+    const totalClassifications =
+      <StyledText
+        additionalStyles={[styles.totalClassifications]}
+        text={`${this.props.user.totalClassifications} total classifications`} />
+
     return (
       <View style={styles.container}>
         <View style={styles.subNavContainer}>
-          <Text style={styles.userName}>
```

Comment:
Can you remove this `Text` component from the RN imports now? I don't believe this component is using it any longer.